### PR TITLE
Windows CI using AppVeyor

### PR DESCRIPTION
**Pull request by nijel**
This is followup to https://github.com/translate/translate/pull/3569 with fixes for the Windows build to make tests pass.

The build now succeeds, see https://ci.appveyor.com/project/nijel/translate/build/1.0.18

I've fixed all failures with exception to oo format, where I was not really sure whether it's test or storage error. Additionally this format doesn't seem to have any upstream docs anymore and is probably dead, so I didn't want to spend much time on that...

To make it usable, registration on AppVeyoyr site is needed and enabling build for this repository. Unfortunately seems to be bound to individual user rather than project there.

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    - PYTHON: "C:\\Python35"
+    - PYTHON: "C:\\Python36"
+    - PYTHON: "C:\\Python36-x64"
+
+install:
+  - "SET PATH=%PYTHON%;%PYTHON%\\Scripts;%PATH%"
+  # We need wheel installed to build wheels
+  - "python.exe -m pip install wheel"
+  # lxml 4.1.1 has binary release for Windows
+  - "python.exe -m pip install lxml==4.1.1"
```
Comment (nschonni) [inline appveyor.yml]: I wasn't sure if there was a better way to do this. Did a quick search and found https://gist.github.com/pombredanne/72130ee6f202e89c13bb but that might be overkill

Comment (nijel) [reply appveyor.yml]: Well it's possible to use this directly in requirements.txt:

```
lxml==4.1.1; platform_system == 'Windows'
```

The question is whether it's better approach than specifying it for tests...

Comment (nschonni) [reply appveyor.yml]: I have been running Weblate on Windows, and I can confirm there is a few hoops like this required. This particular one might be more for avoiding a manual compile that requires a large toolchain and matching Python VC++ tools install

Comment (nijel) [reply appveyor.yml]: Anyway it seems they are consistently doing binary releases now, so it's probably best to remove this completely.

File: translate/convert/test_po2oo.py [hunk 2]
```diff
@@ -49,6 +49,8 @@ def check_roundtrip(self, oosource):
 
     def test_convertoo(self):
         """checks that the convertoo function is working"""
+        if os.name == 'nt':
+            skip("test or storage broken on Windows")
```
Comment (unho) [inline translate/convert/test_po2oo.py]: Better use `skipif`. Don't worry, I will apply this change myself after me merging this PR.

**Comment by unho** [comment]
Great contribution @nschonni and @nijel. I've merged it to `master`. Since PR was not on top of latest `master` I have to manually close this PR:

**Comment by unho** [comment]
@nijel Regarding OO format we should probably drop it in the future since it is a format that won't come back, it will surely become a burden to maintain it, and people needing to convert it will still be able to use older TTK versions.

**Pull request closed without merging**